app/locales/zh-tw/translations.js

```diff
@@ -344,7 +344,8 @@ export default {
       people_count_warning: "您必須填寫一個數字於此欄",
       need_description: "需要的原因",
       need_description_info: "請簡單填寫為什麼需要這些物品",
-      description_warning: "此欄不能留空"
+      description_warning: "此欄不能留空",
+      error_message: "You must select the district"
```

Comment:
You can ask for Chinese translation's of this on volunteers channel and update in the same PR. :)